src/react-components/room/ChatSidebar.js

```diff
@@ -351,6 +371,7 @@ export function ChatSidebar({ onClose, children, ...rest }) {
       title={<FormattedMessage id="chat-sidebar.title" defaultMessage="Chat" />}
       beforeTitle={<CloseButton onClick={onClose} />}
       contentClassName={styles.content}
+      disableOverflowScroll
```

Comment:
Is this actually still needed now? The message list and chat box should both now scroll themselves, so not sure when the container would ever end up overflowing. Not really a problem to explicitly disable it as well, but if it is needed it probably means there is a bug...

Reply:
I kept this inside Sidebar, because in other views like People/Object menus that use the sidebar (not chat), I assume we'll want to enable overflow scroll. Chat is a special case, where the contents manage the scroll themselves.